tests/stubtest_third_party.py

```diff
@@ -43,7 +43,15 @@ def run_stubtest(dist: Path, *, verbose: bool = False, specified_stubs_only: boo
 
     with tempfile.TemporaryDirectory() as tmp:
         venv_dir = Path(tmp)
-        venv.create(venv_dir, with_pip=True, clear=True)
+        try:
+            venv.create(venv_dir, with_pip=True, clear=True)
+        except subprocess.CalledProcessError as e:
+            msg = (
+                "Failed to create virtual environment. Note that on some linux systems, you may need to install the `python3-venv`"
+                " package via your system package manager."
+            )
+            print_command_failure(msg, e)
+            return False
```

Comment:
It might make mores sense to terminate early here -- if you're running stubtest on multiple packages, there's not much point continuing on to the next one if creating a `venv` failed for the first one

```suggestion
            sys.exit(2)
```

Reply:
Good point, would `sys.exit(e.returncode)` be a better option in this case?

Reply:
> Good point, would `sys.exit(e.returncode)` be a better option in this case?

I don't have a very strong opinion, but it looks like the returncode here is `1`. I think I'd rather go for `2` here, since `1` usually indicates for a test that "the test failed", and this isn't that -- this is "the test errored". But, again, no strong opinion. It shouldn't 0 though :p